### ISSUE TITLE
Alter request to physical region is forbidden

### ISSUE THREAD

**Issue by killme2008** [GreptimeTeam/greptimedb#5042]
### What type of bug is this?

Unexpected error, User Experience

### What subsystems are affected?

Storage Engine

### Minimal reproduce step

Try to change a physical table `ttl`:

```sql
mysql> ALTER TABLE greptime_physical_table set ttl='2years';
ERROR 1235 (42000): (Unsupported): Alter request to physical region is forbidden
```

But it fails.

### What did you expect to see?

Success to alter

### What did you see instead?

Throws error.

### What operating system did you use?

All

### What version of GreptimeDB did you use?

0.10.0

**Comment by killme2008** [comment]
Actually, we don't need to forbid this operation for physical tables at all.

**Comment by evenyag** [comment]
Can we close this issue now?

**Comment by killme2008** [comment]
Yes, fixed via #5046 , thansk to @lyang24
